apps/nextjs-website/react-components/components/Form/Form.tsx

```diff
@@ -327,53 +330,35 @@ const Form = ({
         *Campo obbligatorio
       </Typography>
       <Button
-        variant="contained"
+        variant='contained'
         sx={{ width: { md: 'auto', xs: '100%' }, zIndex: 4 }}
         onClick={handleSubmit}
         color={theme === 'dark' ? 'negative' : 'primary'}
       >
         Iscriviti
       </Button>
       <Typography
-        variant="body2"
-        fontWeight="bold"
-        sx={{ mt: 2, position: 'relative', zIndex: 3, color: textColor }}
-      >
-        Inserendo il tuo indirizzo email stai accettando la nostra informativa
-        sul trattamento dei dati personali per la newsletter.
-      </Typography>
-      <Typography
-        variant="body2"
-        sx={{ mt: 2, position: 'relative', zIndex: 3, color: graylinkColor }}
-      >
-        Form protetto tramite reCAPTCHA e Google{' '}
-        <Link
-          href="https://policies.google.com/privacy"
-          sx={{
-            color: graylinkColor,
-            textDecorationColor: graylinkColor,
-            textDecoration: 'underline',
-          }}
-        >
-          {' '}
-          Privacy Policy{' '}
-        </Link>{' '}
-        e{' '}
-        <Link
-          href="https://policies.google.com/terms"
-          sx={{
+        variant='body2'
```

Comment:
```suggestion
        component='div'
        variant='body2'
```
Prevent hydration errors.